lib/rubocop/cop/rspec/pending_without_reason.rb

```diff
@@ -87,35 +87,38 @@ class PendingWithoutReason < Base
           (send #rspec? {#ExampleGroups.all #Examples.all} ... {<(sym :skip) ...> (hash <(pair (sym :skip) true) ...>)})
         PATTERN
 
+        # @!method without_reason?(node)
+        def_node_matcher :without_reason?, <<~PATTERN
+          (send nil? ${:pending :skip})
+        PATTERN
+
         def on_send(node)
           if pending_without_reason?(node)
             add_offense(node, message: 'Give the reason for pending.')
           elsif skipped_without_reason?(node)
             add_offense(node, message: 'Give the reason for skip.')
+          elsif inside_example?(node)
+            without_reason?(node) do |method|
```

Comment:
Performance wise we may want to first check if it's a pending or skip method, e.g. 
```
elsif %I[pending skip].include?(node.method_name) && inside_example?(node)

Reply:
Actually you don't need the without_reason?(node) to return the method name, as it's just node.method_name
So it could be
```
elsif without_reason?(node) && inside_example?(node)
  add_offense(node, message: "Give the reason for #{node.method_name}.")
```

Reply:
Thank you very much. That certainly looks good. I updated this PR.